util/run_tests_version_change

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+if [ ! -f util/.serverpod_util_root ]; then
+    echo "Run this script from the root of the Serverpod repository"
+    echo "I.e. util/run_tests"
```

Comment:
```suggestion
    echo "I.e. util/run_tests_version_change"
```

Though checking the other files, a lot of them refer to some other script (which likely served as the basis for the current one), so we might just fix them all :)